Scheduler/observing_blocks_maintel/BLOCK-252.json

```diff
@@ -0,0 +1,277 @@
+{
+    "name": "BLOCK-252",
+    "program": "BLOCK-252",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/mtrotator/move_rotator.py",
+            "standard": true,
+            "program": "$program",
+            "reason": "SITCOM-1215",
+            "parameters": {
+                "angle": -90
```

Comment:
There is a known issue with the Rotator that causes it to go to a fault near its extremes. We should avoid +/- 90º. I recommend changing this to +/- 85º or even +/- 80º until this issue is solved.